packages/flutter/lib/src/material/overscroll_indicator.dart

```diff
@@ -226,10 +226,11 @@ class _OverscrollIndicatorState extends State<OverscrollIndicator> {
       child: new AnimatedBuilder(
         animation: _extentAnimation,
         builder: (BuildContext context, Widget child) {
-          if (_scrollDirection == null) // Haven't seen a scroll yet.
-            return child;
+          // We keep the same widget hierarchy here, even when we're not
+          // painting anything, to avoid rebuilding the children. This
+          // fixes https://github.com/flutter/flutter/issues/4597 .
```

Comment:
i wouldn't bother with the link to the closed bug. if people want the link they can always find it by looking at the history.
If there's something to say here we should say it inline, rather than point to the issue tracker.